include/swift/SILOptimizer/Utils/SILInliner.h

```diff
@@ -36,108 +36,58 @@ enum class InlineCost : unsigned {
 /// disappear at the LLVM IR level are assigned a cost of 'Free'.
 InlineCost instructionInlineCost(SILInstruction &I);
 
-class SILInliner : public TypeSubstCloner<SILInliner, SILOptFunctionBuilder> {
-  friend class SILInstructionVisitor<SILInliner>;
-  friend class SILCloner<SILInliner>;
-  using SuperTy = TypeSubstCloner<SILInliner, SILOptFunctionBuilder>;
-
+class SILInliner {
```

Comment:
I believe the #include "...TypeSubstCloner.h" is no longer needed in this file.

Reply:
Yeah thanks. I did some #include minimization here:
https://github.com/apple/swift/pull/19786/commits/c781d78782e9da7272b475cba3fced6410cf6df1